lib/v2/screens/authentication/recover/recover_account_found/recover_account_found_screen.dart

```diff
@@ -23,67 +26,86 @@ class RecoverAccountFoundScreen extends StatelessWidget {
         appBar: AppBar(title: const Text("Recover Account")),
         body: BlocBuilder<RecoverAccountFoundBloc, RecoverAccountFoundState>(
           builder: (context, state) {
-            return Padding(
-              padding: const EdgeInsets.all(8.0),
-              child: Column(
-                children: [
-                  Padding(
-                    padding: const EdgeInsets.only(left: 8, right: 8, top: 8),
-                    child: TextFormFieldCustom(
-                      enabled: false,
-                      labelText: 'Link to Activate Key Guardians',
-                      suffixIcon: IconButton(
-                        icon: const Icon(
-                          Icons.copy,
-                          color: AppColors.white,
-                        ),
-                        onPressed: () {
-                          Clipboard.setData(ClipboardData(text: state.linkToActivateGuardians)).then(
-                            (value) {
-                              SnackBarInfo("Copied", ScaffoldMessenger.of(context)).show();
-                            },
-                          );
-                        },
-                      ),
-                      controller: TextEditingController(text: state.linkToActivateGuardians),
-                    ),
-                  ),
-                  Padding(
-                    padding: const EdgeInsets.all(16),
-                    child: Row(
-                      children: [
-                        Text("TODO", style: Theme.of(context).textTheme.button1),
-                        Text("/TODO", style: Theme.of(context).textTheme.button1),
-                        const SizedBox(
-                          width: 24,
-                        ),
-                        Flexible(
-                          child: Text(
-                            "Guardians have accepted your request to recover your account",
-                            style: Theme.of(context).textTheme.buttonLowEmphasis,
+            switch (state.pageState) {
```

Comment:
Unused callback create: (BuildContext context) ------> create: (_)

Reply:
ahhh i wish we could lint this! fixed